src/telemetry.rs

```diff
@@ -1,8 +1,12 @@
+use tracing_subscriber::fmt::format::FmtSpan;
 use tracing_subscriber::prelude::*;
 use tracing_subscriber::{fmt, EnvFilter};
 
 pub fn init_tracing() {
-    let fmt_layer = fmt::layer().with_target(false);
+    let mut fmt_layer = fmt::layer();
+    if std::env::var("INCLUDE_SPAN_EVENTS").is_ok_and(|value| value.eq_ignore_ascii_case("true")) {
```

Comment:
https://www.cs.brandeis.edu/~cs146a/rust/doc-02-21-2015/std/primitive.bool.html#examples

It seems like it would be better this way?

Reply:
it becomes very verbose, the shortest I could get with `parse` is 
```
    let var = std::env::var("INCLUDE_SPAN_EVENTS").unwrap_or_else(|_| "false".to_string());
    if var.parse().is_ok_and(identity) {...}
```
the other somewhat (in)sane way is to just check if the environment variable has been set and ignore the actual value. Should be fine since it's for debugging anyway